payments/vipps/seamless-view.md

```diff
@@ -405,6 +316,68 @@ Content-Type: application/json
 | └➔&nbsp;`isOperational`      | `bool`    | `true` if the transaction is operational; otherwise `false`.                                                                                                                                                 |
 | └➔&nbsp;`operations`         | `array`   | The array of operations that are possible to perform on the transaction in its current state.                                                                                                                |
 
+## Vipps Seamless View integration flow
+
+The sequence diagram below shows the two requests you have to send to
+Swedbank Pay to make a purchase.
+The links will take you directly to the API description for the specific
+request.
+
+```mermaid
+sequenceDiagram
+    Browser->>Merchant: start purchase (pay with VIPPS)
+    activate Merchant
+    Merchant->>-SwedbankPay: POST /psp/vipps/payments ①
+    activate SwedbankPay
+    note left of Merchant: First API request
+    SwedbankPay-->>-Merchant: rel: view-payment ②
+    activate Merchant
+    Merchant-->>-Browser: authorization page
+    activate Browser
+    note left of Browser: Open iframe ③
+    Browser->>Browser: Enter mobile number ④
+    Browser-->>-SwedbankPay: Passing data for authorization
+    activate SwedbankPay
+
+    SwedbankPay-->>-Vipps.API: POST <rel:create-auhtorization> ⑤
+    activate Vipps.API
+    Vipps.API-->>-SwedbankPay: response
+    activate SwedbankPay
+    SwedbankPay-->>-Browser: Authorization response (State=AwaitingActivity) ⑥
+    activate Browser
+    note left of Browser: check your phone
+
+    Vipps.API-->>Vipps_App: Confirm Payment UI
+    activate Vipps_App
+    note left of Vipps.API: Dialogue with Vipps ⑦
+    Vipps_App-->>Vipps_App: Confirmation Dialogue
+    Vipps_App-->>-Vipps.API: Confirmation
+    activate Vipps.API
+    Vipps.API-->>-SwedbankPay: make payment
+    activate SwedbankPay
+    SwedbankPay-->>SwedbankPay: execute payment
+    SwedbankPay-->>-Vipps.API: response
+    activate Vipps.API
+    Vipps.API-->>-SwedbankPay: authorize result
+    activate SwedbankPay
+    SwedbankPay-->>-Browser: Display authorize result
+```
+
+1. When the payer starts the purchase process, you make a `POST` request
+  towards Swedbank Pay with the collected Purchase information.
+1. `rel: view-payment` is a value in one of the operations, sent as a response
+  from Swedbank Pay to the Merchant.
+1. `Open iframe` creates the Swedbank Pay hosted iframe.
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [[ordered-list-marker-value] Marker should be `3`, was `1`](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5278140)